src/Raven.Client/Documents/Session/Loaders/MultiLoaderWithInclude.cs

```diff
@@ -67,14 +68,52 @@ public ILoaderWithInclude<T> Include<TInclude>(Expression<Func<T, IEnumerable<st
             return Include(IncludesUtil.GetPrefixedIncludePath<TInclude>(path.ToPropertyPath(), _session.Conventions));
         }
 
+        /// <summary>
+        /// Includes a counter of the document to be loaded, by counter name
+        /// <param name="name">Name of the counter to include.</param>
+        /// </summary>
+        public ILoaderWithInclude<T> IncludeCounter(string name)
+        {
+            if (_counters == null)
+                _counters = new List<string>();
+            _counters.Add(name);
+            return this;
+        }
+
+        /// <summary>
+        /// Includes counters of the document to be loaded, by counters names
+        /// <param name="names">Names of the counters to include.</param>
+        /// </summary>
+        public ILoaderWithInclude<T> IncludeCounters(string[] names)
+        {
+            if (_counters == null)
+                _counters = new List<string>();
+            if (names != null)
+            {
+                foreach (var name in names)
```

Comment:
Let's assert if this is not null or whitespace.

Reply:
done